serverless/amplify-schema.graphql

```diff
@@ -580,6 +580,7 @@ type Match
         ownerField: "owners"
         operations: [create, update, delete, read]
       }
+      { allow: private, operations: [read] }
```

Comment:
Was this a consistency addition or an unblocking step? I understand what this piece allows as permissions, but can you briefly explain how this factors into the final result?